packages/client-utils/test/sync-tools.test.js

```diff
@@ -403,3 +415,181 @@ test.serial('wait until invitation recevied', async t => {
 
   await t.notThrowsAsync(waitP);
 });
+
+test.serial('wait election result: question handle is adhered', async t => {
```

Comment:
'adhered' is the wrong word, but I'm not sure what you were trying to say. Maybe 'respected' would work? (or verified?)
(and later, too)

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/commit/228bf388dad60f5d798eb25fffffb29e04867a83